Algorithm.java

```diff
@@ -0,0 +1,95 @@
+package K_means;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;

```

Comment:
```suggestion
import java.io.IOException;
import java.util.ArrayList;
import java.util.Scanner;
```suggestion
import java.io.IOException;
import java.util.ArrayList;
import java.util.Scanner;
```suggestion
import java.io.IOException;
import java.util.ArrayList;
import java.util.Scanner;
```suggestion
import java.io.IOException;
import java.util.ArrayList;
import java.util.Scanner;
```suggestion
import java.io.IOException;
import java.util.ArrayList;
import java.util.Scanner;
232323232323223232323
```
```
```
```
```